reservations/notifications.py

```diff
@@ -27,10 +31,23 @@ class Labels:
     NotificationType.WINTER_STORAGE_RESERVATION_CREATED.label,
 )
 
-dummy_context.context.update(
+berth_reservation = BerthReservationFactory.build()
+harbor_choices = HarborChoiceFactory.build_batch(size=3, reservation=berth_reservation)
+winter_storage_reservation = WinterStorageReservationFactory.build()
+winter_area_choices = WinterAreaChoiceFactory.build_batch(
+    size=3, reservation=winter_storage_reservation
+)
+
+dummy_context.update(
     {
-        "created_at": localize_datetime(timezone.now()),
-        # TODO: pass more appropriate in-memory objects
-        "reservation": MagicMock(),
+        COMMON_CONTEXT: {"created_at": localize_datetime(timezone.now())},
+        NotificationType.BERTH_RESERVATION_CREATED: {
+            "reservation": berth_reservation,
+            "harbor_choices": sorted(harbor_choices, key=lambda c: c.priority),
+        },
+        NotificationType.WINTER_STORAGE_RESERVATION_CREATED: {
+            "reservation": WinterStorageReservationFactory.build(),
```

Comment:
Was this supposed to use `winter_storage_reservation` created above?

Reply:
very good catch 😅 👁